tasks/7_random-fraction/random-fraction.js

```diff
@@ -0,0 +1,5 @@
+function randomFraction() {
+  return +Math.random().toFixed(1);
```

Comment:
Please use `Number(val)` to cast a value to a number, it clearly shows your intent, and isn't